### DISCUSSION TITLE
Next NuGet Package release?

### DISCUSSION THREAD

**Discussion by JosephHarvey-Xamarin** [xamarin/XamarinCommunityToolkit#1993] [Q&A]
Hello Team, 

Is there any plan to release an update to the nuget package? The last release was back in May. I know focus is probably on MAUI at this point, but there are still bug fixes that have been made since the last release and I could really use them in my app right now.

Current I'm trying to get some use out of the Tabview, but this bug [Bug] TabViewItem Font does not load on iOS #1978 is causing me grief. There has been a fix for it, but it is not in the current NuGet, nor is it in the Nightly Build (also hasn't been updated since July). I'm also experiencing another issue on iOS with the TabView, which I'll post in the issues thread, but if the NuGets arnt going to up officially updated then I'll have to try another approach to TavView.

Best Regards
Joe

**Comment by bijington** [comment]
We should definitely setup another release if we have unreleased changes. @jfversluis do you have the keys to do that?
